ts/routes/deck-options/FsrsOptions.svelte

```diff
@@ -64,7 +51,7 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
     let computingParams = false;
     let checkingParams = false;
     let computingRetention = false;
-    let simulating = false;
+    const simulating = false;
```

Comment:
Why define this here, instead of passing it in directly as an argument below? In fact, why pass it as an argument at all, instead of just defining it in SimulatorModal?